SEC/Equities/EquitiesExampleIndividuals.rdf

```diff
@@ -130,8 +769,108 @@
 		<fibo-sec-sec-lst:hasHomeExchange rdf:resource="&fibo-fbc-fct-usmkt;NewYorkStockExchange"/>
 	</owl:NamedIndividual>
 	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;XLOMListedAppleIncCommonStock">
+		<rdf:type rdf:resource="&fibo-sec-sec-lst;Listing"/>
+		<rdfs:label>XLOM-listed Apple Inc. common stock</rdfs:label>
+		<skos:definition>Apple Inc. common share listed in the London Stock Exchange</skos:definition>
+		<fibo-fnd-acc-cur:hasCurrency rdf:resource="&fibo-fnd-acc-4217;USDollar"/>
+		<fibo-sec-sec-lst:hasIssue rdf:resource="&fibo-sec-eq-eqind;AppleIncCommonStock"/>
+		<fibo-sec-sec-lst:isTradedOn rdf:resource="&fibo-fbc-fct-mkti;Exchange-XLOM"/>
+		<fibo-sec-sec-lst:lists rdf:resource="&fibo-sec-eq-eqind;AppleIncCommonStock"/>
```

Comment:
lists and hasIssue seem redundant. Would they ever differ?

Reply:
I don't know, which is why I didn't remove it.

Reply:
Raised #871